.nvmrc

```diff
@@ -0,0 +1 @@
+10.17.0
```

Comment:
Let's not fix to any specific node version, I'm personally using v12.13.0. At best we should run the CI on a few node version but that should be CI only and not in the project

Reply:
Ok this can be done with matrices on the Github Action config file, I'll take care of it just give me the list of OS / Node version that you want to be tested on.

Reply:
I would like to follow the releases of node https://nodejs.org/en/about/releases/ and select all the actives especially the LTS:
- v10 nice to have
- v12 (LTS) required
- v13 (latest) nice to have

Reply:
What about the OS?
Ubuntu, Windows, MacOS

Reply:
That would be absolutely fantastic to test on all if it's possible, otherwise, just ubuntu would be great